builtins.go

```diff
@@ -664,6 +665,16 @@ func builtinMd5(i *interpreter, trace traceElement, x value) (value, error) {
 	return makeValueString(hex.EncodeToString(hash[:])), nil
 }
 
+func builtinBase64(i *interpreter, trace traceElement, input value) (value, error) {
+	str, err := i.getString(input, trace)
```

Comment:
This should also work with arrays  of bytes, see: https://github.com/google/jsonnet/blob/552d8ec6f6b973a6357b83eb9bacd707366d28f0/stdlib/std.jsonnet#L1072

The string version should error out when there are characters with codepoints > 255 and those <= should use codepoints-as-bytes encoding (the current implementation results in UTF8 encoding, which would be a better option, if we didn't need to worry about backwards compatibility). See: https://github.com/google/jsonnet/blob/552d8ec6f6b973a6357b83eb9bacd707366d28f0/stdlib/std.jsonnet#L1108

And we should have tests to cover all that :-).

Reply:
Thanks for the feedback. I'll add that functionality and add tests as well.

Reply:
done

Reply:
implemented for arrays and strings